src/option.h

```diff
@@ -514,6 +514,8 @@ EXTERN int	p_confirm;	// 'confirm'
 #endif
 EXTERN int	p_cp;		// 'compatible'
 EXTERN char_u	*p_cot;		// 'completeopt'
+EXTERN char_u	*p_cia;		// 'completeitemalign'
+EXTERN unsigned cia_flags;	// order flags of 'completeitemalign'
 EXTERN unsigned	cot_flags;	// flags from 'completeopt'
```

Comment:
The order of variables here may be a bit strange: `p_cot` comes before `p_cia` but `cot_flags` comes after `cia_flags`.

Reply:
right. move to top of p_cot